src/Aspire.Dashboard/Model/ResourceLogLine.cs

```diff
@@ -0,0 +1,8 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Dashboard.Model;
+
+public readonly record struct ResourceLogLine(int LineNumber, string Content, bool IsErrorMessage)
```

Comment:
Should this have the same doc comments as the LogLine in Hosting?

Reply:
LogLine only has public comments because it's a public API.

This one is just for us. The names are self explanatory.